Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.yml

```diff
@@ -52,6 +52,7 @@ configuration:
   required: false
 - display: Fetch incident with specific states.
   name: states
+  defaultvalue: new,open,assigned,closed,ignored
```

Comment:
I think it won't be applied to existing instances that get the update, need to update the python as well